rehack_tests/stdlib/stdlib.cma.php/Genlex.php

```diff
@@ -319,83 +319,120 @@ private static function load($joo_global_object) {
+                    case 62:
+                    // FALLTHROUGH
+                    case 63:
+                    // FALLTHROUGH
+                    case 64:
+                      $switch__0 = 2;
+                      break;
+                    // FALLTHROUGH
+                    default:
+                      $switch__0 = 0;
                     }
-                    throw $runtime["caml_wrap_thrown_exception"](Vector{0, $Stream[2], $cst__0}) as \Throwable;
-                  }
-                  else if ($switcher === 40) {
-                    $call1($Stream[12], $strm);
-                    if ($counter < 50) {
-                      $counter__0 = (int) ($counter + 1);
-                      return $maybe_comment->contents($counter__0, $strm);
-                    }
-                    return $caml_trampoline_return(
-                      $maybe_comment->contents,
-                      varray[0,$strm]
-                    );
-                  }
-                  else if ($switcher === 45) {
-                    $call1($Stream[12], $strm);return $neg_number($strm);
-                  }
-                  else if ($switcher === 9) {}
-                  else if ($switcher === 10) {}
-                  else if ($switcher === 12) {}
-                  else if ($switcher === 13) {}
-                  else if ($switcher === 26) {}
-                  else if ($switcher === 32) {
-                    $call1($Stream[12], $strm);continue;
-                  }
-                  else if ($switcher === 48) {}
-                  else if ($switcher === 49) {}
-                  else if ($switcher === 50) {}
-                  else if ($switcher === 51) {}
-                  else if ($switcher === 52) {}
-                  else if ($switcher === 53) {}
-                  else if ($switcher === 54) {}
-                  else if ($switcher === 55) {}
-                  else if ($switcher === 56) {}
-                  else if ($switcher === 57) {
-                    $call1($Stream[12], $strm);
-                    $reset_buffer(0);
-                    $store($l);
-                    return $number($strm);
-                  }
-                  else if ($switcher === 33) {}
-                  else if ($switcher === 35) {}
-                  else if ($switcher === 36) {}
-                  else if ($switcher === 37) {}
-                  else if ($switcher === 38) {}
-                  else if ($switcher === 42) {}
-                  else if ($switcher === 43) {}
-                  else if ($switcher === 47) {}
-                  else if ($switcher === 58) {}
-                  else if ($switcher === 60) {}
-                  else if ($switcher === 61) {}
-                  else if ($switcher === 62) {}
-                  else if ($switcher === 63) {}
-                  else if ($switcher === 64) {$switch__0 = 2;break;
-                  }
-                  else {$switch__0 = 0;}
+                  if ($continue_label === "#") {continue;}
```

Comment:
I couldn't easily reason about how this diff would handle the following cases:

```
a:
for() {
  switch() {
    continue a;
  }
}


b:
for() {
  a:
  for() {
    switch() {
      continue a;
    }
  }
}


  a:
  for() {
    switch() {
     // Both unlabeled and labeled.
      continue;
      continue a;
    }
  }
```

Reply:
Those cases will be output like this:


```
for() {
  $continue_label = null;
  switch() {
    $continue_label = "a";
    break;
  }
  if ($continue_label === "a") {
    continue;
  } 
}


for() {
  $continue_label = null;
  for() {
    $continue_label = null;
    switch() {
      $continue_label = "b";
      break;
    }
    if ($continue_label != null) {
      break;
    }
  }
  if ($continue_label === "b") {
    continue;
  }
}


  for() {
    $continue_label = null;
    switch() {
      // unlabeled
      $continue_label = "#";
      break;
      // labeled
      $continue_label = "a";
      continue a;
    }
    if ($continue_label === "#") {
      continue;
    } else if ($continue_label === "a") {
      continue;
    }
  }
```

Reply:
Nice!